docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,314 @@
+? Default script ConfigMap name
+? brigade.js file path relative to the repository root
+? Upload a default brigade.js script brigade.js
+? Secret for the Generic Gateway (alphanumeric characters only). Press Enter if you want it to be auto-generated mysecret
+Project ID: brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac
+```
+
+*Yes, we do know that the process to create a non-VCS/Generic Gateway Project is a bit cumbersome. This is going to be fixed in [#816](https://github.com/brigadecore/brigade/issues/816)*
+
+### Send a SimpleEvent to the Generic Gateway
+
+Next, we should send a simple JSON object to test our Generic Gateway. By default, Generic Gateway is not exposed to the outside world, since it is created as a [Cluster IP Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+To send traffic to a Cluster IP Service, we can use `kubectl port-forward`. The following command will open a tunnel from local port 8081 to port 8081 on the Generic Gateway Service.
+
+```bash
+kubectl port-forward service/brigade-brigade-generic-gateway 8081:8081
+```
+
+To send a JSON message to Generic Gateway, open a new shell and try this curl command:
+
+```bash
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{
+    "key1": "value1",
+    "key2": "value2"
+}' \
+  http://localhost:8081/simpleevents/v1/brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac/mysecret
+```
+
+Pay attention to the Generic Gateway URL for a SimpleEvent, it is `HOST:PORT/simpleevents/v1/Project-ID/genericGatewaySecret`
+
+If all is well, you'll get back a 200 result: 
+
+```json
+{"status":"Success. Build created"}
+```
+
+### View this Build's log
+
+Let's now see what happened in our Worker Pods:
+
+```bash
+kubectl get pods
+```
+
+You'll see that two Pods were created for this Build. The worker Pod and the `echosimpleevent` Pod.
+
+```
+NAME                                               READY   STATUS      RESTARTS   AGE
+brigade-brigade-api-6c5d6f4dcb-fzqg5               1/1     Running     0          22m
+brigade-brigade-ctrl-6cc46c6769-mm2v7              1/1     Running     0          22m
+brigade-brigade-generic-gateway-6f6496958f-sms5q   1/1     Running     0          2m51s
+brigade-kashti-6bf64b8458-vhqxb                    1/1     Running     0          22m
+brigade-worker-01d6tzxyqwbym5qdxa8wj4s6mx          0/1     Completed   0          111s
+echosimpleevent-01d6tzxyqwbym5qdxa8wj4s6mx         0/1     Completed   0          16s
+```
+
+Let's get the `echosimpleevent` logs:
+
+```bash
+kubectl logs echosimpleevent-01d6tzxyqwbym5qdxa8wj4s6mx
+```
+
+You'll see the output that we requested in our Brigade.js file:
+
+```
+Project github.com/deis/empty-testbed
+event type: simpleevent
+payload {\n    "key1": "value1",\n    "key2": "value2"\n}
+```
+
+To learn more about the Generic Gateway, check our docs [here](https://docs.brigade.sh/topics/genericgateway/).
+
+## Kashti and Vacuum
```

Comment:
nit: maybe just split up into 2 sections, one for `Kashti` and the other for `Brigade Vacuum`?  Having them paired is slightly confusing as they aren't necessarily really related to each other.

Reply:
Great suggestion. To tell you the truth, I'd rather have a separate section on the docs about vacuum that could potentially contain information like the one in #846